source/PlayState.hx

```diff
@@ -2,17 +2,25 @@ package;
 
 import flixel.FlxG;
 import flixel.FlxState;
+import flixel.tweens.FlxTween;
+import flixel.tweens.FlxEase;
 import flixel.graphics.FlxGraphic;
 import flixel.tile.FlxTilemap;
 import flixel.util.FlxColor;
 import flixel.group.FlxGroup;
 
+
 class PlayState extends FlxState
 {
 	public var GRAVITY(default, never):Float = 600;
 
 	private var map:FlxTilemap;
 	private var player:Player;
+	private var _grpTrap:FlxTypedGroup<Trap>;
+	private var trap:Trap;
+	private var trap1:Trap;
+	private var trap2:Trap;
+	private var trap3:Trap;
```

Comment:
Maybe rename these slightly? The variable names make it sound like there are four separate traps, instead of the four pieces of the bar.